rust/lance/src/io/exec/testing.rs

```diff
@@ -69,6 +75,10 @@ impl ExecutionPlan for TestingExec {
     }
 
     fn statistics(&self) -> datafusion::error::Result<datafusion::physical_plan::Statistics> {
-        todo!()
```

Comment:
Note: DF now actually calls `statistics` and so we can't get away with `todo!()` here anymore.